presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotFilterExpressionConverter.java

```diff
@@ -201,6 +201,23 @@ else if (timeFieldExpression instanceof ConstantExpression) {
         return Optional.of(timeValueString);
     }
 
+    private PinotExpression handleContains(
+            CallExpression contains,
+            Function<VariableReferenceExpression, Selection> context)
+    {
+        if (contains.getArguments().size() == 2) {
+            RowExpression left = contains.getArguments().get(0);
+            RowExpression right = contains.getArguments().get(1);
```

Comment:
Should one side be a literal here ? What happens if I do a: array_col1 contains array_col2 ? 

Would presto ensure that array_col2 is a literal ?

What happens when array_col2 is a non literal scalar ? Would that be supported by Pinot ?

Reply:
The BuiltInFunctionNamespaceHandleResolver will handle the type check.
It will throw exception if the second argument is an array.

Reply:
The builtin array functions has strict type check: https://prestodb.io/docs/current/functions/array.html

Reply:
E.g. 
```
presto:default> explain select sum(array_sum(divwheelsons)) from airlinestats WHERE contains(divwheelsons, divwheelsoffs);
Query 20201004_011726_00000_xdccd failed: line 1:69: Unexpected parameters (array(integer), array(integer)) for function contains. Expected: contains(array(T), T) T:comparable
explain select sum(array_sum(divwheelsons)) from airlinestats WHERE contains(divwheelsons, divwheelsoffs)
```

Reply:
So should one of the arguments be a literal here ?

Reply:
right, arg[1] should be a literal

Reply:
added a check on arg[1] to ensure it's a ConstantExpression.